pkg/auth/providerrefresh/refresh_daemon.go

```diff
@@ -58,6 +58,10 @@ func StartRefreshDaemon(ctx context.Context, scaledContext *config.ScaledContext
 }
 
 func UpdateRefreshCronTime(refreshCronTime string) {
+	if ref == nil {
+		return
+	}
+
 	c.Stop()
```

Comment:
This currently will drop the current cron if the input is bad, a validator before we get here would be good and don't drop if the user manages to change through `kubectl` and the input is bad